lib/api/core/httpRouter/index.js

```diff
@@ -24,19 +24,27 @@ function Router(kuzzle) {
   this.kuzzle = kuzzle;
 
   this.defaultHeaders = {
-    'Content-Type': 'application/json',
+    'content-type': 'application/json',
```

Comment:
It has no impact since headers are case insensitive. I lowercased it in the HTTP router default headers so that it can be directly compared with headers handled by the `RequestResponse` object, greatly simplifying unit tests.